.github/workflows/build-containers.yml

```diff
@@ -61,134 +52,235 @@ jobs:
                                            -wholename ./{} \
                                            -printf "%P\n")
           else
-               # List all image root dirs. Example value:
-               # "opensciencegrid/vo-frontend opensciencegrid/ospool-cm"
                images=$(find $ORG_DIR -mindepth 1 \
                                       -maxdepth 1 \
                                       -type d \
                                       -printf "$ORG_DIR/%P\n")
           fi
-
-          # Ensure that the generated JSON array has a member,
-          # otherwise GHA will throw an error about an empty matrix
-          # vector in subsequent steps
           image_json=$(echo -n "${images:-dummy}" | jq -Rcs '.|split("\n")')
           echo "json=$image_json" >> $GITHUB_OUTPUT
 
-  build:
+  check-config:
     runs-on: ubuntu-latest
-    needs: build-image-list
-    if: needs.build-image-list.outputs.images != '["dummy"]'
-    # Prevent a single build failure from killing the workflow.
-    # This is safe since subsequent pushes should fail on cache load
-    continue-on-error: true
```

Comment:
It looks like you completely got rid of this logic and I believe we still need it.